connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkConnector.kt

```diff
@@ -23,7 +26,52 @@ class UplynkConnector(
     private val theoplayerView: THEOplayerView,
 ) {
     private lateinit var integration: UplynkAdIntegration
-    private val eventDispatcher = UplynkEventDispatcher()
+
+    /**
+     * Scheduled ad breaks
+     */
+    var adBreaks: List<UplynkAdBreak>? = null
+        private set
+
+    /**
+     * The ad break that is currently played or `null` otherwise
+     */
+    var currentAdBreak: UplynkAdBreak? = null
+        private set
+
+    /**
+     * The ad that is currently played or `null` otherwise
+     */
+    var currentAd: UplynkAd? = null
+        private set
+
+    private val eventDispatcher = UplynkEventDispatcher().also {
+        it.addListener(object : UplynkListener {
```

Comment:
Nitpick: make `UplynkConnector` implementing `UplynkListener`

Reply:
I thought about it, but didn't do it because `UplynkConnector` is a part of public API and I think it might be confusing for a user of this class. So I decided to do it like this.

Do you think it would be better to make implement `UplynkConnector` implementing `UplynkListener`?

Reply:
Indeed, it would be confusing..
Maybe we move the checks to the internal too?